test/e2e/framework/test_context.go

```diff
@@ -335,6 +337,7 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 	// koordinator configs
 	flags.StringVar(&TestContext.KoordinatorComponentNamespace, "koordinator-component-namespace", "koordinator-system", "The namespace of the koordinator components deployed to.")
 	flags.StringVar(&TestContext.SLOCtrlConfigMap, "slo-config-name", "slo-controller-config", "The name of the slo-controller configmap.")
+	flags.StringVar(&TestContext.KoordSchedulerName, "koord-scheduler-name", "default-scheduler", "The SchedulerName of the koord-scheduler.")
```

Comment:
The schedulerName we reveal to the outside world has always been koord-scheduler. So I recommend keeping the default value of koord-scheduler. In addition, some cases in test/e2e/scheduling have koord-scheduler written in them. Can they be corrected together?

Reply:
fixed